packages/fulfillment/src/migrations/Migration20240221115805.ts

```diff
@@ -1,6 +1,6 @@
 import { Migration } from '@mikro-orm/migrations';
 
-export class Migration20240219115644 extends Migration {
+export class Migration20240221115805 extends Migration {
```

Comment:
It's nice to name migrations so they are more easily discoverable (at least when we have more than 1)

Reply:
the name is auto generated, do you have strong opinion to rename it since it is the only one?

Reply:
@adrien2p I know it's autogenerated, but all of them are named the same, so when the second one comes we'll need to rename it then :) Not a big deal at all, I just thought it's nicer if this is called eg `InitialSetup...` for example

Reply:
ahah got it 👍

Reply:
I ve changed my script option so that everytime I generate the initial migration it will have a specific name